capz/readme.md

```diff
@@ -1,4 +1,5 @@
 These scripts and templates are used with https://github.com/kubernetes-sigs/cluster-api-provider-azure and https://github.com/Azure/azure-capi-cli-extension.  
+For azure-capi-cli-extension to work ensure you have installed az cli on the machine that runs the script: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli  
```

Comment:
could you give an example here?  something like "for instance, if you run into `someerror` when running on wsl you may need to make sure azure cli is install in wsl.

Reply:
Surely!